src/Evolution/Particles/MonteCarlo/TemplatedLocalFunctions.hpp

```diff
@@ -24,12 +24,36 @@ struct Packet;
 
 namespace detail {
 
+const double proton_mass_MeV = 938.272046;
```

Comment:
I made a request in #5903 to move this to the Units.hpp file; just want to make sure it doesn't get add here